README.md

```diff
@@ -56,6 +58,16 @@ Once it's activated, the extension will automatically trigger itself whenever yo
 
 ![screenshot](https://user-images.githubusercontent.com/38117856/107463541-542e8500-6b2c-11eb-8b25-082f344c1587.png)
 
+### Bookmarklet
+
+The online tool can be used as a [bookmarklet](https://en.wikipedia.org/wiki/Bookmarklet). To add one manually, press `Ctrl+D` to create a new bookmark and add this code in the URL field:
+
+```js
+javascript:!function(){if(m=window.location.href.match(/github.com\/([^\/]+)\/([^\/]+)/),m){window.open(`https://useful-forks.github.io/?repo=${m[1]}/${m[2]}`)}else window.alert("Not a GitHub repo")}();
```

Comment:
```suggestion
javascript:!function(){if(m=window.location.href.match(/github\.com\/([\w.-]+)\/([\w.-]+)/),m){window.open(`https://useful-forks.github.io/?repo=${m[1]}/${m[2]}`)}else window.alert("Not a GitHub repo")}();
```

1. The `.` was not escaped, so it was matching "any character"
2. The 2nd capture group allowed matching on `something?foo=bar` when we'd really only want `something`
3. It's safer to work with an inclusive sets of [the allowed characters](https://stackoverflow.com/a/59082561/9768291), rather than excluding some

Note that `\w` is equivalent to `[a-zA-Z0-9_]`